src/test/java/uk/gov/hmcts/reform/professionalapi/controller/constants/ErrorConstantsTest.java

```diff
@@ -1,21 +1,20 @@
-package uk.gov.hmcts.reform.professionalapi.controller;
-
-import static org.assertj.core.api.Assertions.assertThat;
-
-import org.junit.Test;
-import uk.gov.hmcts.reform.professionalapi.controller.advice.ErrorConstants;
-
-public class ErrorConstantsTest {
-
-    @Test
-    public void check_constants() {
-
-        assertThat(ErrorConstants.MALFORMED_JSON.getErrorMessage()).isEqualTo("1 : Malformed Input Request");
-
-        assertThat(ErrorConstants.UNSUPPORTED_MEDIA_TYPES.getErrorMessage()).isEqualTo("2 : Unsupported Media Type");
-
-        assertThat(ErrorConstants.INVALID_REQUEST.getErrorMessage()).isEqualTo("3 : There is a problem with your request. Please check and try again");
-
-    }
-
-}
+package uk.gov.hmcts.reform.professionalapi.controller.constants;
+
+import static org.assertj.core.api.Assertions.assertThat;
+
+import org.junit.Test;
+
+public class ErrorConstantsTest {
+
+    @Test
+    public void check_constants() {
+
+        assertThat(ErrorConstants.MALFORMED_JSON.getErrorMessage()).isEqualTo("1 : Malformed Input Request");
+
+        assertThat(ErrorConstants.UNSUPPORTED_MEDIA_TYPES.getErrorMessage()).isEqualTo("2 : Unsupported Media Type");
+
+        assertThat(ErrorConstants.INVALID_REQUEST.getErrorMessage()).isEqualTo("3 : There is a problem with your request. Please check and try again");
+
+    }
```

Comment:
Needs to cover all error messages , but not this way, We have tested enums in UP by considering the behaviour.

Reply:
These Enums are used in _ExceptionMapper_, _CcdErrorMessageResolver_, _UserProfileUpdateRequestValidatorImpl_ - which all have test coverage.

 I can remove the direct tests for these constants - same as we did in UP.

Reply:
This has been addressed in: https://github.com/hmcts/rd-professional-api/pull/450/commits/1b9dfd59e89e2556173983349ef504330f3b92f0